adafruit_as726x.py

```diff
@@ -30,71 +30,78 @@
 import time
 from adafruit_bus_device.i2c_device import I2CDevice
 from micropython import const
+import busio
```

Comment:
I said it in the other PR, just reviewing this so it doesn't get merged before you can move this to the try/except typing block!

Reply:
Moved the busio into the try/except block. Thanks for reviewing!